### PR TITLE
feat: notify user about configuration file not exist and provide inst…

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
…ruction

E2E TEST: NA

### CODE REVIEW

File: docs/fx-core/spfx-help.md [hunk 1]
```diff
@@ -116,4 +117,39 @@ Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
 
 ```sh
 npm install @microsoft/generator-sharepoint@1.14 -g
-```
\ No newline at end of file
+```
+
+## 3. Failure to add additional SPFx tab<a name="addfeature"></a>
+
+Multi-tab SPFx project is supported in Teams Toolkit. To add an additional SPFx tab, you can try "Add features" and select "SPFx tab". Behind the scenes, Yeoman generator is executed according to configuration file(.yo-rc.json) in current solution.
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:124]: Missing a couple of `the` and space before parenthesis: `....file (.yo-rc.json)`:
"... Yeoman Generator is executed according to the configuration file (.yo-rc.json) in the current solution."

File: docs/fx-core/spfx-help.md [hunk 2]
```diff
@@ -116,4 +117,39 @@ Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
...
+
+If .yo-rc.json file doesn't exist in your SPFx project, adding SPFx tab might fail with potential issues.
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:126]: If .yo-rc.json file doesn't exist in your SPFx project, adding SPFx tab will fail with the following error:

File: docs/fx-core/spfx-help.md [hunk 3]
```diff
@@ -116,4 +117,39 @@ Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
...
+## 3. Failure to add additional SPFx tab<a name="addfeature"></a>
+
+Multi-tab SPFx project is supported in Teams Toolkit. To add an additional SPFx tab, you can try "Add features" and select "SPFx tab". Behind the scenes, Yeoman generator is executed according to configuration file(.yo-rc.json) in current solution.
+
+If .yo-rc.json file doesn't exist in your SPFx project, adding SPFx tab might fail with potential issues. 
+
+#### SPFx.NoConfigurationFile
+![image](../images/fx-core/spfx/spfx-no-configuration-file.png)
+
+Please follow the instructions here to continue:
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:131]: ### Remediation

Please follow the instructions here to continue:

**Review by xu-hong — COMMENTED** [review]
Thanks for putting these together. I agree with the overall content, added some minor tweaks on words.

File: docs/fx-core/spfx-help.md (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply docs/fx-core/spfx-help.md]: Updated

**Review by jayzhang — APPROVED** [review]

**Pull request merged**